staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -628,20 +628,26 @@ func (r *Request) tryThrottleWithInfo(ctx context.Context, retryInfo string) err
 	latency := time.Since(now)
 
 	var message string
+	var kv []any
+	if retryInfo == "" {
+		retryInfo = "client-side throttling, not priority and fairness"
+	}
```

Comment:
I think I meant to replace that switch because I wasn't happy with that approach, but didn't finish. Now it's simpler:

```Go
	if latency > longThrottleLatency {
		if retryInfo == "" {
			retryInfo = "client-side throttling, not priority and fairness"
		}
		klog.FromContext(ctx).V(3).Info("Waited before sending request", "delay", latency, "reason", retryInfo, "verb", r.verb, "URL", r.URL()}
	}
```

Reply:
Pushed: https://github.com/kubernetes/kubernetes/compare/f97e9e1d83bcdc87a1d6ef0766453975a43a35ba..6456dd6c73c3bbe15c1c8cdf85054908f212962e